.github/workflows/static.yml

```
@@ -0,0 +1,29 @@
+name: Static Analysis
+
+on: [ push, pull_request ]
+
+jobs:
+  analysis:
+    name: larastan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)